openfisca_us/parameters/gov/states/in/tax/income/exemptions/additional/amount.yaml

```
@@ -0,0 +1,11 @@
+description: Additional exemption amount for qualifying dependent children in IN.
+values:
+  2021-01-01: 1_500
+metadata:
+  unit: currency-USD
+  period: year
+  name: in_exemptions_additional_amount_dependent_child
+  label: IN exemptions additional amount for dependent childen
+  reference:
+    - title: IC 6-3-1-3.5	"Adjusted gross income" (a)(4)(A)
```

Unfortunately the tab issue will break the package in Colab, even though it doesn't cause test failures here. See https://github.com/PolicyEngine/openfisca-us/issues/976.
```suggestion
    - title: IC 6-3-1-3.5 "Adjusted gross income" (a)(4)(A)
```